src/runway/path.py

```diff
@@ -0,0 +1,130 @@
+"""Runway configuration 'path' settings."""
+# pylint: disable=unused-import
+from typing import Optional, Dict, List, Tuple  # noqa: F401
+
+import os
+import logging
+import six
+
+from .sources.git import Git
+
+LOGGER = logging.getLogger('runway')
+
+
+class Path(object):  # pylint: disable=too-many-instance-attributes
+    """Runway configuration 'path' settings object."""
+
+    def __init__(self, module, env_root, cache_dir=None, git_source_class=Git):
+        # type: (str, str, Optional[str])-> Path
+        """Initialize."""
```

Comment:
add documentation for `Args` (minus `self` or `cls`) - in other functions/methods as well.